handbook/brand.md

```diff
@@ -297,6 +297,32 @@ In Figma:
   * Avoid using SVGs or icon fonts.
 3. Click the __Export__ button.
 
+## Request Digital Experience services
+
+The following table details how to request reviews and art assets from the Digital Experience team.
```

Comment:
Let's make this about how to publish content or changes to content, rather than how to request reviews.

Reply:
@mikermcneil, are you referring to the wording of this intro only, or the entire table?

Reply:
@mike-j-thomas i.e. Tim doesn't say "I want to publish some ad artwork today".  He says "I want to run an ad campaign".

I would also differentiate between "New handbook page" and "Handbook edits", since I think a new handbook page is gated by digital experience, whereas handbook edits are edited post facto.

Reply:
@mikermcneil, ok. I'll have another go on Monday and remove the artwork/service references. 

It's clear now that I don't exactly understand what this table should include.  
It would be helpful to see an example of one row. I can't find anything in the wild to model it off.

Reply:
> It would be helpful to see an example of one row. I can't find anything in the wild to model it off.

<img width="1234" alt="image" src="https://user-images.githubusercontent.com/618009/171942129-31d3a1bc-17f3-4d0a-9285-4fb912a98104.png">
<img width="1102" alt="image" src="https://user-images.githubusercontent.com/618009/171942223-a9405cf7-6ddb-48de-9812-7e813ebafaf7.png">
<img width="1107" alt="image" src="https://user-images.githubusercontent.com/618009/171942273-86975b17-d018-404f-962b-923d6e8c818e.png">

Reply:
The screenshots above include an example row about how to publish a tweet.

Reply:
<img width="642" alt="image" src="https://user-images.githubusercontent.com/618009/171942633-a1829755-6b54-4960-b96a-ce627355e689.png">

Reply:
@mikermcneil - ah, I'm sorry. I thought I'd bookmarked that [thread](https://fleetdm.slack.com/archives/C01GQUZ91TN/p1653978058024329) to read, but I lost it in Slack. Thanks again for sharing it.

Reply:
@mikermcneil, I committed another attempt, ready for review.